src/main/java/frc/robot/Constants.java

```diff
@@ -16,4 +16,7 @@
  * constants are needed, to reduce verbosity.
  */
 public final class Constants {
+    public static final class EndgameActuatorConstants {
+        public static final int ACTUATOR_SOLENOID = 0;
```

Comment:
Double solenoids require two ports, see `EndgameActuator.java`. I recommend calling them `ACTUATOR_SOLENOID_1` and `ACTUATOR_SOLENOID_2`.